datacontract/schemas/datacontract-1.1.0.schema.json

```diff
@@ -1212,11 +1212,48 @@
         },
         "delimiter": {
           "type": "string",
-          "enum": [
-            "new_line",
-            "array"
+          "anyOf": [
+            { "enum": ["new_line", "array"] },
+            { "pattern": "^.$" }
           ],
-          "description": "Only for format = json. How multiple json documents are delimited within one file"
+          "description": "For JSON format, only 'new_line' or 'array' is allowed to indicate how multiple JSON documents are delimited. For CSV format, any single character can be used as the delimiter between columns. Only valid for CSV."
```

Comment:
the master for the schema is here: https://github.com/datacontract/datacontract-specification/

Reply:
@jochenchrist should I not modify this one and just change the one in the other repo then?

Reply:
Please don't the file here, but propose a change in the other repo.

We still can continue to use the provided keys in the data_contract_specification (as custom extensions)